packages/react/src/components/timeInput/TimeInput.module.scss

```diff
@@ -0,0 +1,82 @@
+@import "../../styles/common";
+
+.timeInputFrame {
+  display: inline-flex;
+  align-items: center;
+  padding: 0 calc(var(--spacing-s) - 3px);
+
+  &.disabled {
+    background-color: var(--input-background-disabled);
+    border-color: var(--input-border-color-disabled);
+    color: var(--input-color-disabled);
+    cursor: not-allowed;
+
+    &:hover {
+      border-color: var(--input-border-color-disabled);
+    }
+  }
+
+  // TODO: It would be better to make text-input's focus styles more flexible. But it requires changes to core styles which might have unwanted results.
+  &:not([readonly]):focus {
+    box-shadow: 0 0 0 var(--outline-width) var(--focus-outline-color);
+    transform: translate3d(0, 0, 0);
+  }
+}
+
+.fullInput {
+  position: absolute;
+  width: 0;
+  height: 0;
+  overflow: hidden;
+  opacity: 0;
+  pointer-events: none;
+}
+
+.partialInputLabel {
+  @extend %hiddenFromScreen;
+}
+
+.partialInput, .divider {
+  border: 0;
+  padding: 0;
+  -webkit-appearance: none;
+  box-sizing: border-box;
+  font-size: 1.125rem;
```

Comment:
Do we have a font-size variable for this?

Reply:
`var(--fontsize-body-l) ` is the same but I do not know if there is something more suitable 🤔

Reply:
I guess it would be better to have these as fixed values 🤔